packages/flutter_tools/lib/src/project.dart

```diff
@@ -377,11 +382,13 @@ class AndroidProject {
 
   Future<void> ensureReadyForPlatformSpecificTooling() async {
     if (isModule && _shouldRegenerateFromTemplate()) {
-      _regenerateLibrary();
+      await _regenerateLibrary();
       // Add ephemeral host app, if an editable host app does not already exist.
       if (!_editableHostAppDirectory.existsSync()) {
-        _overwriteFromTemplate(fs.path.join('module', 'android', 'host_app_common'), _ephemeralDirectory);
-        _overwriteFromTemplate(fs.path.join('module', 'android', 'host_app_ephemeral'), _ephemeralDirectory);
+        print('Generating host_app_common template for Android');
```

Comment:
Instead of `print()` calls here and below, we should use `printTrace()` or `printStatus()`.

Reply:
From the context of the messages, it looks like `printTrace()` would be right.